### PR TITLE
SPA for `<authConfig>` and `<roleConfig>`

### PR DESCRIPTION
**Pull request by bdpiparva**

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  var AuthConfigModalBody = {
+    controller: function (args) {
+      var ctrl          = this;
+      var setPluginInfo = function (pluginInfo) {
+        args.pluginInfo(pluginInfo);
+        args.saveDisabled(!pluginInfo);
+      };
+
+      this.selectAuthConfigId = function () {
+        var pluginInfo = _.find(PluginInfos(), function (pi) {
```
Comment (iamsumanth) [inline server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx]: Can we rename ``pi`` to something like ``pluginInfo`` ?

File: server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      };
+
+      this.clearMessage = function () {
+        args.message({});
+      };
+
+      this.getMessage = function () {
+        if (args.message && args.message() && args.message().type) {
+          return (
+            <f.row>
```
Comment (iamsumanth) [inline server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx]: is it possible to refactor this ? somehow returning "view (html part of the code)" from controller method looks odd to me.

File: server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx (hunk 1 above)
Comment (bdpiparva) [inline server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx]: Done

File: server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx (hunk 2 above)
Comment (bdpiparva) [inline server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/auth_configs/auth_config_modal_body.js.msx]: done

**Comment by bdpiparva** [comment]
Will send new PR with mithril 1.0

**Pull request closed without merging**